ostelco-ios-client/Extensions/UIViewController+StartApplePay.swift

```diff
@@ -9,6 +9,7 @@
 import Stripe
 import Siesta
 
+
```

Comment:
SwiftLint should be yelling at you about this but: Excess newline

Reply:
Done.